docs/core/admin/migration.md

```diff
@@ -410,3 +410,36 @@ with GMail migration. It will:
     ```sh
     doveadm backup -a 'virtual/All' -O '-$GmailHaveLabels' -R -u user@domain imapc:
     ```
+
+## Optimizing Synchronization
+
+The `dsync -s` parameter can be used to significantly improve performance for incremental migrations.
```

Comment:
I'd about using `dsync` here and elsewhere, especially in monospace font. There is no dsync command anymore. It could be okay to use "dsync `-s`" here or maybe `doveadm sync/backup -s`